src/Raven.Server/Documents/Replication/DocumentReplicationLoader.cs

```diff
@@ -341,6 +344,13 @@ private void OnOutgoingSendingFailed(OutgoingReplicationHandler instance, Except
                 ConnectionFailureInfo failureInfo;
                 if (_outgoingFailureInfo.TryGetValue(instance.Destination, out failureInfo) == false)
                     return;
+                
+                failureInfo.DestinationDbId = instance.DestinationDbId;
+                failureInfo.LastHeartbeatTicks = instance.LastHeartbeatTicks;
+                failureInfo.LastSentDocumentEtag = instance._lastSentDocumentEtag;
+                failureInfo.LastSentIndexOrTransformerEtag = instance._lastSentIndexOrTransformerEtag;
+                using (instance._documentsContext.OpenReadTransaction())
+                    failureInfo.GlobalChangeVector = instance._database.DocumentsStorage.GetDatabaseChangeVector(instance._documentsContext);
```

Comment:
Huh?! You are reading here the _local_ change vector, on the failing entry value, why?